Mathlib/Analysis/Distribution/SchwartzSpace.lean

```diff
@@ -1025,63 +1026,75 @@ section Integration
 
 open Real Complex Filter MeasureTheory MeasureTheory.Measure FiniteDimensional
 
-variable [NormedAddCommGroup D] [NormedSpace ℝ D] [NormedAddCommGroup V] [NormedSpace ℂ V]
-variable [MeasurableSpace D] [BorelSpace D] [FiniteDimensional ℝ D]
+variable [IsROrC 𝕜]
+variable [NormedAddCommGroup D] [NormedSpace ℝ D]
+variable [NormedAddCommGroup V] [NormedSpace ℝ V] [NormedSpace 𝕜 V]
+variable [MeasurableSpace D] [BorelSpace D] [SecondCountableTopology D]
+
+section
+
+variable {μ : Measure D} {n : ℕ} (h : Integrable (fun x ↦ (1 + ‖x‖) ^ (- (n : ℝ))) μ)
```

Comment:
Could you move `h` to the theorems that need it? Having Prop variables around makes it really hard to grasp what theorems are saying.